docs/api.md

```diff
@@ -53,7 +53,7 @@ ReactDOM.render(
 
 Connects a React component to a Redux store.
 
-It does not modify the component class passed to it.  
+It does not modify the component class passed to it.
```

Comment:
This was probably your editor being helpful, but the trailing whitespace here was to create a line break in the rendered markdown, and should be put back.